docs/Windows-VisualStudio.md

```diff
@@ -1,27 +1,24 @@
 # Visual Studio 2019 <!-- omit in toc -->
 
-You can use Visual Studio 2019 to examine the samples (compile them and run them in an SDL window) or to build your own apps for the 32blit API.
+You can use Visual Studio 2019 to examine the examples (compile and run them on Windows using the SDL HAL) or to build your own apps for 32blit.
 
 See [Building & Running On 32Blit](32blit.md) if you want to compile examples/projects to run on 32Blit.
 
 - [Requirements](#requirements)
 - [Option 1: Use the solution file](#option-1-use-the-solution-file)
+- [Option 2: Use Visual Studio's built-in CMake support (Recommended)](#option-2-use-visual-studios-built-in-cmake-support-recommended)
   - [Get started with your own game](#get-started-with-your-own-game)
-- [Option 2: Use Visual Studio's built-in CMake support](#option-2-use-visual-studios-built-in-cmake-support)
   - [Building your own game](#building-your-own-game)
   - [Building for 32Blit](#building-for-32blit)
 - [Troubleshooting](#troubleshooting)
+  - [Cannot open include file: 'SDL.h': No such file or directory](#cannot-open-include-file-sdlh-no-such-file-or-directory)
+  - [`cmd.exe` not recognised](#cmdexe-not-recognised)
 
 ## Requirements
 
 You will need Visual Studio 2019 (preferably version 16.4).
 
-Make sure you install C++ desktop development support.
-
-You will also need to download SDL2 development libraries from the [SDL homepage](https://www.libsdl.org/download-2.0.php). Here find the latest version of the VC development libraries (at the time of this writing SDL2-devel-2.0.10-VC.zip). Additionally, download SDL2_image from [here](https://www.libsdl.org/projects/SDL_image/) (SDL2_image-devel-2.0.5-VC.zip) and SDL2_net from [here](https://www.libsdl.org/projects/SDL_net/) (SDL2_net-devel-2.0.1-VC.zip).
-
-Place these in the `vs\sdl\` folder. You will need to merge the include/lib directories. If you are using CMake/Open Folder, these are downloaded automatically.
-
+Make sure you install "Desktop development with C++". If you plan to build for 33blit, you'll also need "Linux development with C++", making sure "C++ CMake tools for Linux" and "Embedded and IoT development tools" are selected in the "Optional" section.
```

Comment:
Ah, the new improved 33blit :smile:

Reply:
One more bit is nothing to sniff at! Though memory alignment of smaller types is quite complicated so you're encouraged to use `uint33_t` everywhere.